.github/workflows/release-sisyphos.yml

```
@@ -102,7 +102,7 @@ jobs:
     uses: ./.github/workflows/_70_post_release.yml
     with:
       network: "sisyphos"
-  upload-engine-shared-libs:
-    needs: [build, build-m2]
+  upload-versioned-build-artifacts:
+    needs: [upgrade-check]
```

```suggestion
    needs: [upgrade-check, build, build-m2]
```
Upgrade-check doesn't depend on build or build-m2, which are the steps that are required to complete before this job will succeed